.gitignore

```diff
@@ -47,3 +47,6 @@
 
 #Doxygen docs
 /docs/
+
+# Our configuration file
+streaming.conf
```

Comment:
We're "ignoring" this file, but it's still showing up in the diff. Is this intentional?

Reply:
Yes. I want the initial configuration to be added to the repo but any changes folks make should be ignored in commits.